### PR TITLE
Tweak: Updated the content in nested tabs [ED-8546]

### PR DESCRIPTION
**Pull request by hein-obox**

### CODE REVIEW

File: modules/tabs-v2/widgets/tabs-v2.php [hunk 1]
```diff
@@ -158,10 +158,11 @@ protected function register_controls() {
 				],
 			],
 			'title_field' => '{{{ tab_title }}}',
+			'button_text' => 'Add Tab',
 		] );
 
-		$this->add_responsive_control( 'tabs_position', [
-			'label' => esc_html__( 'Position', 'elementor' ),
+		$this->add_responsive_control( 'tabs_direction', [
```
Comment (davseve) [inline modules/tabs-v2/widgets/tabs-v2.php:164]: I noticed that the order of the Direction buttons has changed between Aviv's video to what appears in this branch.
[Avivs video](https://nimb.ws/wc6Ure):
[Current branch](https://nimb.ws/pscJWC)

Comment (hein-obox) [reply modules/tabs-v2/widgets/tabs-v2.php:164]: @davseve 
Can you please double check on your side if it works correctly?

Comment (davseve) [reply modules/tabs-v2/widgets/tabs-v2.php:164]: It works as it should, only the order changed.

Comment (hein-obox) [reply modules/tabs-v2/widgets/tabs-v2.php:164]: I have checked it. It seems to work correctly.
I think we continue to PR Review 2.

**Review by davseve — APPROVED** [review]

**Pull request merged**